kava-stats.js

```diff
@@ -390,9 +394,20 @@ const mapUsdxLimits = async (denoms, cdpParamsData) => {
     coins[denom] = { debtLimit: limit }
   }
 
+      for (const denom of cdpParamsData.collateral_params) {
+        const secondsPerYear = 31536000;
+        const stabilityFeePercentage = ((Number(denom.stability_fee) ** secondsPerYear - 1) * 100).toFixed(2);
```

Comment:
Since you loop through stability fee above at line 378, you shouldn't need to loop through it again here.